### PR TITLE
Update Sprites to be drawn from center.

### PR DESCRIPTION
**Pull request by jessetilro**
Sprites are now drawn at their center point instead of their top left
corner. The SpriteStore now reads and the Sprite stores the dimensions of
the Sprite image.

Issue #226

<feature,organisation

### CODE REVIEW

File: src/main/java/nl/tudelft/scrumbledore/sprite/SpriteStore.java [hunk 1]
```diff
@@ -126,7 +131,17 @@ private Sprite getSpriteFromFile(File file, String dir) {
     int pos = name.lastIndexOf('.');
     String id = name.substring(0, pos);
     String ext = name.substring(pos + 1);
-    return new Sprite(id, ext, dir);
+
+    Vector size = new Vector(0, 0);
```
Comment (NielsWarnars) [inline src/main/java/nl/tudelft/scrumbledore/sprite/SpriteStore.java]: This one can be placed directly in return new Sprite(id, ext, dir, size);
<review

Comment (dalderliesten) [inline src/main/java/nl/tudelft/scrumbledore/sprite/SpriteStore.java]: Agreed with Niels.

<review

**Comment by NielsWarnars** [comment]
No further CheckStyle/FindBugs/PMD issues

File: src/main/java/nl/tudelft/scrumbledore/sprite/SpriteStore.java [hunk 2]
```diff
@@ -126,7 +131,17 @@ private Sprite getSpriteFromFile(File file, String dir) {
...
+    try {
+      BufferedImage sprite = ImageIO.read(file);
+      size.setX(sprite.getWidth());
+      size.setY(sprite.getHeight());
+    } catch (IOException e) {
+      e.printStackTrace();
+    }
+
+    return new Sprite(id, ext, dir, size);
```
Comment (NielsWarnars) [inline src/main/java/nl/tudelft/scrumbledore/sprite/SpriteStore.java]: size vector can be directly inserted here
<review

File: src/main/java/nl/tudelft/scrumbledore/sprite/SpriteStore.java (hunk 1 above)
Comment (jessetilro) [inline src/main/java/nl/tudelft/scrumbledore/sprite/SpriteStore.java]: No it can't, because it is written to in the try statement.

**Pull request merged**